src/NuGet.Core/NuGet.Packaging/RuntimeModel/RuntimeGraph.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace NuGet.RuntimeModel
 {
-    public class RuntimeGraph : IEquatable<RuntimeGraph>
+    public sealed class RuntimeGraph : IEquatable<RuntimeGraph>
```

Comment:
Note, this is a breaking change - but given how NuGet objects implement Clone where they assume no derived classes, this is probably what should have been the behavior.